modules/mobile/docs/openapi.yaml

```diff
@@ -3099,6 +3099,44 @@ paths:
       security:
         - Bearer: []
       summary: /v0/messaging/health/recipients
+  /v0/messaging/health/threads/{id}/move:
+    patch:
+      description:
+        Moves all received messages in a secure message thread to a specified folder. Sent
+        messages, draft messages, and deleted messages will not be moved.
+      operationId: post_move_thread
+      parameters:
+        - description: The id of the thread that is to be moved
+          in: path
+          name: id
+          required: true
+          schema:
+            type: string
+        - description: The id of the folder that the message is to be moved to
+          in: query
+          name: folder_id,
+          required: true
+          schema:
+            type: string
+        - $ref: '#/components/parameters/InflectionHeader'
+      responses:
+        '204':
+          description: The message thread was moved successfully
+        '401':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: Unauthorized
+        '403':
+          content:
+            application/json:
+              schema:
+                $ref: ./schemas/Errors.yml
+          description: Not authorized for access to secure messages
```

Comment:
Are there other possible errors? I would imagine there's an error code for when you're attempting to move a non-existent thread or when you're trying to move a thread to a non-existent folder

Reply:
Wow I'm surprised/not surprised there's no documentation on what it will return with a folder id or thread not being found

Reply:
I was surprised as well but I didn't know how to go about finding the response if thread is not found. I'll check staging and will update specs